root/static/scripts/release-editor/validation.js

```diff
@@ -123,13 +125,36 @@ $(function () {
 });
 
 
+// Search releases with the same barcode
+function searchExistingBarcode(field, barcode) {
+    request({
+        url: `/ws/2/release/?query=barcode:${barcode}&fmt=json`
```

Comment:
I couldn't find a way to run this precise query with /ws/js/, so I chose /ws/2/

Reply:
Nevermind, I found the helper function for /ws/2/ requests in the release-editor

Reply:
@mwiencek: is there any way of doing this with ws/js? I guess we do use ws/2 for stuff in the RE already, so maybe this is expected.